querybook/webapp/ui/ColumnIcon/ColumnIcon.tsx

```diff
@@ -0,0 +1,33 @@
+import clsx from 'clsx';
+import React from 'react';
+
+import { Icon } from 'ui/Icon/Icon';
+import { AllLucideIconNames } from 'ui/Icon/LucideIcons';
+
+export interface IColumnIconProps {
+    name?: AllLucideIconNames;
+    tooltip?: string;
+    hollow?: boolean;
```

Comment:
can you use fill instead? or just pass props from this component to the Icon component? so there is no flip of variables